package.json

```diff
@@ -61,7 +61,9 @@
     "axios": "0.21.4",
     "ejs": "^3.1.7",
     "handlebars": "4.7.7",
-    "node-notifier": "8.0.1",
-    "eslint-plugin-import/**/json5": "^1.0.2"
+    "node-notifier": "8.0.1"
+  },
```

Comment:
yarn keeps trying to remove `"eslint-plugin-import/**/json5": "^1.0.2"` so I obeyed it but we can remove this change